crates/shared/src/price_estimation.rs

```diff
@@ -214,6 +216,14 @@ pub struct Arguments {
     #[clap(long, env, default_value = "solve")]
     pub yearn_solver_path: String,
 
+    /// The API endpoint to call the Raven solver for price estimation
+    #[clap(long, env)]
+    pub raven_solver_url: Option<Url>,
+   
+    /// The API path to use for solving.
+    #[clap(long, env, default_value = "solve")]
+    pub raven_solver_path: String,
```

Comment:
Tbh, I am not sure if this is needed,  and whether additional changes are needed because of that, as the solving part is already taken care of somewhere else (as Raven already participates in price estimation).

Reply:
In general yes, let's keep it